.github/workflows/check-fixup-commits.yml

```
@@ -0,0 +1,38 @@
+name: Check for fixup commits
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - converted_to_draft
+      - ready_for_review
+      - labeled
+      - unlabeled
+      - auto_merge_enabled
+      - auto_merge_disabled
+
+jobs:
+  check-fixup-commits:
+    runs-on: ubuntu-latest
+
```

Let's only run this in case a rebase strategy has been chosen to avoid spamming people with failing tests notifications.

```suggestion
    if: >-
      github.event_name == 'pull_request' &&
      github.event.pull_request.draft == false &&
      github.event.pull_request.base.ref == 'master' && 
      contains(github.event.pull_request.labels.*.name, 'automerge:rebase') &&
      !contains(github.event.pull_request.labels.*.name, 'bypass:linear-history')
```